aptos-move/framework/supra-framework/sources/stake.move

```diff
@@ -2795,11 +2795,7 @@ module supra_framework::stake {
             let vci: &ValidatorConsensusInfo = obj;
             validator_consensus_info::get_addr(vci)
         });
-        let vci_pks = vector::map_ref(&vci_vec_0, |obj|{
```

Comment:
Btw, what are those change for?

Reply:
@axiongsupra, removed the `vci_pks` variable, as it appears to be unused.